neqo-transport/src/send_stream.rs

```diff
@@ -280,7 +280,7 @@ pub struct TxBuffer {
 }
 
 impl TxBuffer {
-    const BUFFER_SIZE: usize = 0xFFFF; // 64 KiB
+    pub const BUFFER_SIZE: usize = 0x10_0000; // 1 MiB
```

Comment:
Do you think that it makes sense to use a single common constant for these?

Reply:
I think keeping them separate might come in handy, so let's keep them separate for now.